git_externals/git_externals.py

```diff
@@ -396,39 +396,43 @@ def autosvn_update_checkout(reset):
                 gitext_up(recursive, entries, reset=reset, use_gitsvn=use_gitsvn)
 
 
-def gitext_recursive_info(root_dir):
-    git_exts = load_gitexts()
-    git_exts = {ext_repo: ext for ext_repo, ext in git_exts.items()
+def gitext_recursive_info(root_dir, recursive=True, externals=[]):
+    git_exts = {ext_repo: ext for ext_repo, ext in load_gitexts().items()
```

Comment:
What are we skipping here? Do you think it's better logging to the user or documenting in code what is happening?